.github/workflows/upload-content-files.yml

```diff
@@ -58,15 +82,17 @@ jobs:
         uses: actions/checkout@v2
       - run: git pull -r
       - run: aws s3 cp ./config/forms/forms.json "s3://$BUCKET/forms.json"
+      - run: aws s3 cp ./config/answerFilters/answerFilters.json "s3://$BUCKET/answerFilters.json"
+      - run: aws s3 cp ./config/nextSteps/nextStepOptions.json "s3://$BUCKET/nextStepOptions.json"
 
-  push-forms-to-production:
-    name: 🚀 Push forms to S3 (Production)
+  push-forms-and-filters-to-production:
+    name: 🚀 Push forms and answer filters to S3 (Production)
     runs-on: ubuntu-latest
-    concurrency: content-production
+    concurrency: content-update-forms-and-filters-production
     needs:
-      - push-forms-to-staging
+      - push-content-to-staging
     environment:
-      name: Content - Production
+      name: Content Update Forms And Answer Filters - Production
```

Comment:
You'll want to also create these environments in GitHub and apply the same branch protections to them as currently exist to allow approval by @elsa-bardout and @amygrainbow let me know if you need me to action this.

Reply:
Thanks 🙏 , Yeah I think I'll need your help with this.